### DISCUSSION TITLE
How to undo?

### DISCUSSION THREAD

**Discussion by gsingh93** [cursorless-dev/cursorless#1091] [Q&A] [answered]
Is there a command for undoing the last edit?

**Comment by AndreasArvidsson** [comment, marked as answer]
Today there is no Cursorless undo command but we have an issue for it #317

**Reply by gsingh93** [reply]
Thanks! Is there a way for me to setup a non-phrase level undo? Essentially just bind a word to cmd+z? I've never used Talon before.

**Reply by AndreasArvidsson** [reply]
If you are using Knausj that is already bound to `undo that`
https://github.com/knausj85/knausj_talon/blob/644d330be80bddfe7bfc314a15903185b3111cf7/core/edit/standard.talon#L12

**Reply by AndreasArvidsson** [reply]
For general Talon questions please check have a look in the Talon slack
https://talonvoice.com/chat
